test-packages/e2e-tests/odata.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const cds = require('@sap/cds');
+const express = require('express');
+
+module.exports = {
+  createODataApp() {
```

Comment:
Is the name ODataApp good? I think we refer to this often as `CAP` but I am not sure what is a good name.

Reply:
I have no strong opinion here. I called it odata, because it is the express behind `/odata`.

Reply:
I would call it CapApp but as you like.

Reply:
Ok, done.